### ISSUE TITLE
fix: "unexpected token '>'" at build time after format

### ISSUE THREAD

**Issue by peterpeterparker** [dfinity/prettier-plugin-motoko#29]
Great work with the plugin 👍

I gave it a try in Papyrs backend / providers ([PR 21](https://github.com/papyrs/ic/pull/21)) and landed on following issue at build time after having formatted the code:

> /Users/daviddalbusco/.../storage.store.mo:96.5-96.6: syntax error [M0001], unexpected token '>', expected one of token or <phrase> sequence:
  >

The root cause of the issue is the following type of function:

```
private func update(key: Text, record: Data, putData: PutData): Result.Result<Data, Text> {
```

Gets formatted to:

```
private func checkTimestamp(record : Data, id : Text, updated_at : Time.Time) : Result.Result<
  Text,
  Text
> {
```

The Motoko compiler does not expect the last `>` to finds place on a new line. Adapting code manually as following fix the build issue:

```
private func checkTimestamp(record : Data, id : Text, updated_at : Time.Time) : Result.Result<
  Text,
  Text> {
```

`dfx 0.11.1`

**Comment by rvanasa** [comment]
Thanks for the bug report! I just fixed the issue locally and will include this in the next release (once I wake up tomorrow morning).

**Comment by rvanasa** [comment]
Fixed in version `0.1.1`. Let me know if you run into anything else!

**Comment by peterpeterparker** [comment]
Looks good thanks, I merged the PR introducing the formatter in my `main` branch 🤙
